react/features/overlay/components/UserMediaPermissionsOverlay.js

```diff
@@ -0,0 +1,96 @@
+/* global interfaceConfig */
+import AbstractOverlay from './AbstractOverlay';
+import React from 'react';
+
+/**
+ * Implements a React Component for overlay with guidance how to proceed with
+ * gUM prompt.
+ */
+export default class UserMediaPermissionsOverlay extends AbstractOverlay {
+    /**
+     * UserMediaPermissionsOverlay component's property types.
+     *
+     * @static
+     */
+    static propTypes = {
+        /**
+         * The browser which is used currently. The text is different for every
+         * browser.
+         * @public
+         * @type {string}
+         */
+        browser: React.PropTypes.string
+    }
+
+    /**
+     * Initializes a new SuspendedOverlay instance.
+     *
+     * @param {Object} props - The read-only properties with which the new
+     * instance is to be initialized.
+     * @public
+     */
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            /**
+             * The src value of the image for the policy logo.
+             * @type {string}
+             */
+            policyLogoSrc: interfaceConfig.POLICY_LOGO
```

Comment:
doesn't look like this can ever change throughout the lifetime of the app, thus you can make it an instance variable.